packages/core/src/ui/Layout/Layout.tsx

```diff
@@ -60,166 +84,59 @@ export const Layout: React.FC<Props> = ({
         <meta name="viewport" content="initial-scale=1.0, width=device-width" />
       </Head>
       <div className="min-h-screen bg-background dark:bg-background-dark">
-        {/* TODO logic for picking title */}
-        <Nav
-          title={nav.title}
-          logo={nav.logo}
-          links={nav.links}
-          search={nav.search}
-          social={nav.social}
-          defaultTheme={theme.defaultTheme}
-          themeToggleIcon={theme.themeToggleIcon}
-        />
-        <div className="relative mx-auto">
-          <main className="flex-auto">
+        {/* NAVBAR */}
+        <div
+          className={clsx(
+            "sticky top-0 z-50 w-full",
+            isScrolled
+              ? "dark:bg-background-dark/95 bg-background/95 backdrop-blur [@supports(backdrop-filter:blur(0))]:dark:bg-background-dark/75"
+              : "dark:bg-background-dark bg-background"
+          )}
```

Comment:
why did we move this from nav component to here ?

Reply:
Since now the Layout component controls the positioning of all the main blocks it's composed of,  I've moved paddings and width classes out of Nav (and out of the other child components it's now composed of), so that we can easily adjust e.g. width of the whole site content (for which you need to adjust paddings, margins or width of the main Layout blocks - in a few different places - e.g. Navbar and content below it). And having e.g. max width applied to these components only in the Layout makes it easier, as it's easy to find. So now e.g. Nav component is everything that's inside, i.e. links, search field etc. But we want to apply the background on the whole container. That's why it's moved to Layout. Also, I just think telling if the site has scrolled or not is rather a job of the whole Layout and in this case, it can be used to style also other components, not only the Nav. Or e.g. based on its value we could display an additional component - a button to scroll to the top, or sth like that :) Hope that makes sense 😅

Reply:
Sounds like a plan 😉

Reply:
Now that there are some more paddings in the layout the typed animation in the Hero will lap again, but we can fix that.